cli/cmd.go

```diff
@@ -83,7 +83,7 @@ func envForRepo(t repo.RepoType) string {
 	case repo.FullNode:
 		return "FULLNODE_API_INFO"
 	case repo.StorageMiner:
-		return "STORAGE_API_INFO"
+		return "MINER_API_INFO"
```

Comment:
To fully support deprecation we'd need to figure out how to handle this stuff as well. This only returns the used value (which is fine), but we would need to handle both in the `GetAPIInfo` function where the `os.LookupEnv` occurs.

@magik6k might have some ideas where, but an off the cuff solution would be to change both of these `flagForRepo` and `envForRepo` to return a slice. We use the first element for when we print the values, but search for both when we consume them (`GetAPIInfo`).

Reply:
how about add `flagForRepoDeprecation` and `envForRepoDeprecation` func? And delete them after deprecation period. Then it won't change exist function.

Reply:
Oh yes, that sound like a better idea